app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -186,9 +186,9 @@ You have the right to apply to train to teach in England if you are resident in
 * [the Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)
 * [the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)
 
-### Deadline for applications
+### When to apply for teacher training
 
-Applications for teacher training courses starting in autumn 2023 will be open until September 2023.
+Applications for teacher training open in October for courses starting the following September, or sometimes January, and remain open almost all year. However, it's best to apply as early as you can. 
```

Comment:
I was a little confused by this sentence. Do you mean until the courses start that September? As it might suggest people could still apply after the Sept in for e.g. Oct or Nov?

Reply:
I don't know if **and** sometimes January reads slightly better? But feel free to ignore!

Reply:
Yes it does I was bothered by that too!